pkgs/data/themes/flat-remix-gnome/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchFromGitHub
+, glib
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flat-remix-gnome";
+  version = "20210623";
+
+  src = fetchFromGitHub {
+    owner = "daniruiz";
+    repo = pname;
+    rev = version;
+    hash = "sha256-FKsbAvjhXb2ipe3XqACM6OwGYwbBbzvDjsUQYCIQ8NM=";
+  };
+
+  nativeBuildInputs = [ glib ];
```

Comment:
doesn't glib need to be in buildInputs if there's compiled schemas?

cc @jtojnar

Reply:
No, `glib-compile-schemas` is just a build time dependency. And so is the setup hook.